server/users/routes.js

```diff
@@ -20,14 +18,14 @@ userRouter.get('/', async (req, res) => {
 });
 
 // POST new user to database
-// userRouter.post('/', async (req, res) => {
-//   const user = req.body;
-//   await addUser(user)
-//     .then(() => {
-//       res.status(200).send('add users');
-//     })
-//     .catch((err) => {
-//       console.log(err);
-//       res.status(500).json({ error: 'something went wrong adding user to db.' });
-//     });
-// });
+userRouter.post('/', async (req, res) => {
+  const user = req.body;
+  await addUser(user)
```

Comment:
Nit: so the `await` here doesn't actually do anything, same as above - the `.then()` and the `.catch()` already handle all the effects of the `Promise`